pages/scribbles.tsx

```diff
@@ -12,7 +12,16 @@ const Scribbles = ({ articles }) => {
       pageDescription='Articles and technical blogs about Full Stack Web Development, Open-source and Communities'
     >
       <Container>
-        <Heading>Scribbles</Heading>
+        <CenterElements>
+          <Heading>Scribbles</Heading>
+          <SubHeading>
+          
+            Suffering from Writer’s block? Scribbling around in your book aimlessly with no words flowing?<br/> 
+            We post regular writing prompts as fodder for your inspiration. 
+            Let your creativity flow and mail your works to us at <u>xpress.d361@gmail.com </u>
```

Comment:
@Swatishree-Mahapatra you can use an anchor tag with mailto for the mail

Reply:
done